x-pack/platform/plugins/shared/alerting/common/routes/gaps/apis/find/schemas/v1.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+import { gapsResponseSchemaV1 } from '../../../response';
+
+export const findGapsBodySchema = schema.object(
+  {
+    end: schema.maybe(schema.string()),
```

Comment:
The PR description has `start` and `end` as required. Should these not be `schema.maybe` or is the PR description incorrect?

Reply:
Make it required https://github.com/elastic/kibana/commit/c45c6719bb88e422345043601901125bb2bce8fb